drivers/stepper/adi_tmc/adi_tmc_reg.h

```diff
@@ -21,8 +21,8 @@ extern "C" {
 /** Common Registers for TMC50XX and TMC51XX */
 #if defined(CONFIG_STEPPER_ADI_TMC50XX) || defined(CONFIG_STEPPER_ADI_TMC51XX)
 
-#define TMC5XXX_WRITE_BIT        0x80U
-#define TMC5XXX_ADDRESS_MASK     0x7FU
+#define TMC5XXX_WRITE_BIT    0x80U
+#define TMC5XXX_ADDRESS_MASK 0x7FU
```

Comment:
```
drivers: stepper: adi_tmc: extend rampstat mask range

Now added pos_reached_mask for polling rampstat when
position reached event is not supported, for example,
with uart drivers. Consequently, when the position is
reached, the position is reached, and the event flag is
set active. Hence, an additional value is now introduced.
```
Unrelated whitespace changes are adding a lot of noise to this commit

Reply:
It seems like clang-formatting did this, and has been an issue with this file. 
Would it be fine to fix this formatting issue?